### ISSUE TITLE
`caffe._caffe.Blob()` in pycaffe `RuntimeError: This class cannot be instantiated from Python`!

### ISSUE THREAD

**Issue by thesby** [BVLC/caffe#4982]
I want to use pycaffe to create my PythonLayer. After coding, I just want to test my layer. So I want to create a Blob as the input of  `setup()`. I find that:
`caffe._caffe.Blob()` in pycaffe `RuntimeError: This class cannot be instantiated from Python`!
What's wrong here ?

**Comment by williford** [comment]
Can you try creating an `InputLayer`?

To-do: `InputLayer` needs better documentation. I am planning on adding it to the Layer Catalogue, although it also doesn't have doxygen.

**Comment by thesby** [comment]
Yes, an `InputLayer` works. But why cann't we create a Blob directly?
